OptoTap/README.md

```diff
@@ -35,7 +35,7 @@ This version saves some vertical room by requiring you to solder your connector
 
 <img src="./images/optotap_v2.png" width=600>
 
-This version uses an AP63205WU switch mode regulator to drop input voltage and supply 5V to the sensor. This gives a bit of additional room to add a JST-PH 2.0mm pitch connector as well [part number S3B-PH-K-S (LF)(SN)].
+This version uses an AP63205WU switch mode regulator to drop input voltage and supply 5V to the sensor. This gives a bit of additional room to add a 3-pin JST-PH 2.0mm pitch connector as well [part number S3B-PH-K-S (LF)(SN)].
```

Comment:
I don't *really* see the necessity, as this is redundant with the part number specified.